### PR TITLE
ufs fix

### PR DESCRIPTION
**Pull request by sevki**
fixes #54

### CODE REVIEW

**Comment by elbing** [comment]
We don't want lionkov mess. One thing to free Ron from this is, right, vedorizing it. But I think is killing flies with a gun. Just because git submodule init/update don't point to master branch. Perhaps we included the submodule wrongly.

File: bootstrap.sh [hunk 1]
```diff
@@ -1,17 +1,10 @@
 #!/bin/bash
-if [ -n "$(git remote -v | awk '$1=="origin"{print $2}' | grep gerrithub)" ]; then
-	echo Setting up git...
-	curl -Lso .git/hooks/commit-msg http://review.gerrithub.io/tools/hooks/commit-msg
-	chmod u+x .git/hooks/commit-msg
-	# this is apparently the preferred method and might even work over HTTP
-	# https://gerrit-review.googlesource.com/Documentation/cmd-receive-pack.html
-	git config remote.origin.push HEAD:refs/for/master%r=rminnich@gmail.com,r=crossd@gmail.com,r=elbingmiss@gmail.com,r=anyrhine@gmail.com,r=0intro@gmail.com,r=hdonnay@gmail.com,r=john@jfloren.net,r=rafael.fernandez@taisis.com,r=keith.poole@gmail.com,r=s@sevki.org
-fi
```
Comment (rminnich) [inline bootstrap.sh]: Mission creep. The description of this CL does not include the de-gerritification of it. Please make CLs that have a single action.

Comment (sevki) [inline bootstrap.sh]: I'm sorry. It was right there

Comment (rminnich) [inline bootstrap.sh]: LGTM

**Comment by rminnich** [comment]
LGTM

**Pull request merged**